deployments/typesMillau.json

```diff
@@ -0,0 +1,52 @@
+{
+	"AuthoritySet": {
+		"authorities": "AuthorityList",
+		"set_id": "SetId"
+	},
+	"BridgedBlockHash": "H256",
+	"BridgedBlockHasher": "BlakeTwoAndKeccak256",
+	"BridgedBlockNumber": "u64",
+	"BridgedDigest": {
```

Comment:
I'm pretty curious about this. We don't have a `BridgedDigest` struct anywhere in the code, how did you figure that we needed a definition for it?

Reply:
@tomusdrw , any comments about this?

Reply:
We need to re-define the `BridgedHeader` type, because in Rust it's a generic type `generic::Header<Hash>`. The JS parser doesn't handle generic types at all, so we need separate type for every generic instance (i.e. `generic::Header<H256>` or `generic::Header<H512>`).
The types propagate further down, so since `Digest` is generic as well (and depends on hash size), we must re-define it as well to parse it correctly.